interface/main/tabs/templates/patient_data_template.php

```diff
@@ -46,49 +47,55 @@
             </div>
             </span>
             <span class="patientDataColumn">
-                &nbsp;
                  <!-- ko if: patient -->
                 <!-- ko with: patient -->
-                <div>
-                    <span>Selected Encounter:</span>
+                    <div data-bind="click: clickNewEncounter">
+                    	<a class="css_button_small" title="<?php echo xlt("New Encounter");?>">
+						<i style="font-size:150%;" class="fa fa-plus"></i>
+						</a>
+					</div>
+                <div class="Info">
+                    <span><?php echo xlt("Open Encounter") ?>:</span>
+					<strong>
                     <!-- ko if:selectedEncounter() -->
-                        <span data-bind="text:selectedEncounter().date()"></span>
+                        <span data-bind="text:selectedEncounter().date()"></span>&nbsp;
                         (<span data-bind="text:selectedEncounter().id()"></span>)
                     <!-- /ko -->
                     <!-- ko if:!selectedEncounter() -->
                         <?php echo xlt("None") ?>
                     <!-- /ko -->
+					</strong>
                 </div>
-                <!-- /ko -->
-                <!-- /ko -->
-            </span>
-            <span class="patientDataColumn patientEncountersColumn">
-                <!-- ko if: patient -->
-                <!-- ko with: patient -->
-
-                <span class="patientEncounterList" >
-                    <div data-bind="click: clickNewEncounter"><?php echo xlt("New Encounter");?></div>
-                    <div data-bind="click: clickEncounterList"><?php echo xlt("Past Encounter List");?>
-                        (<span data-bind="text:encounterArray().length"></span>)
-                    </div>
-                    <table class="encounters">
+                <!-- ko if:encounterArray().length > 0 -->
+                <div class="patientInfo">
+                <span class="patientEncounterList">
+					<a class="bolder" data-bind="click: clickEncounterList">
+						<?php echo xlt("View") ?>&nbsp;
+						<label data-bind="text:encounterArray().length"></label>&nbsp;
+                    	<?php echo xlt("past encounter(s)");?>
```

Comment:
This won't work well for translations(breaking it up with the number inside the string) and since is such a core part of the UI, will be better to do:
View Past Encounters (#)